README.md

```diff
@@ -136,7 +136,8 @@ class CloneOptions {
 }
 
 int Main(string[] args) {
-  return CommandLine.Parser.Default.ParseArguments<AddOptions, CommitOptions, CloneOptions>(args)
```

Comment:
There is no `ParseArguments` variant with more that 1 generic parameter. That's the reason I changed that line.